src/pages/instances/forms/UploadInstanceBackupFileForm.tsx

```diff
@@ -107,7 +108,7 @@ const UploadInstanceBackupFileForm: FC<Props> = ({
         toastNotify.info(
           <>
             Upload completed. Now creating instance{" "}
-            <strong>{values.name}</strong>.
+            <ResourceLabel bold type="virtual-machine" value={values.name} />.
```

Comment:
type can be container

Reply:
will resolve with generic `instance` type here